pom.xml

```diff
@@ -134,7 +134,7 @@
                         <configuration>
                             <referenceVersion>2.22ea3</referenceVersion>
                             <artifactsURI>https://teamcity.chronicle.software/repository/download</artifactsURI>
-                            <!--                            <binaryCompatibilityPercentageRequired>92</binaryCompatibilityPercentageRequired>-->
+                            <binaryCompatibilityPercentageRequired>83.6</binaryCompatibilityPercentageRequired>
```

Comment:
why ?

Reply:
See comments below. The checker picks up some changes which it shouldn't.